cors/handler.go

```diff
@@ -0,0 +1,75 @@
+package cors
```

Comment:
Maybe cors/preflight would be better? And then we could add an actual middleware that adds `Access-Control-Allow-Origin` in package cors